model_cards/sagorsarker/bangla-bert-base/README.md

```diff
@@ -0,0 +1,101 @@
+---
+language: bn
+tags:
+- bert
+- bengali
+- bengali-lm
+- bangla
+license: MIT
+datasets:
+- common_crawl
+- wikipedia
+---
```

Comment:
```suggestion
- oscar
---
```

I suggest adding `oscar` as it will be in `datasets` soon and the link will then just work